films/models.py

```diff
@@ -0,0 +1,5 @@
+from django.db import models
+
+
+class Movie:
+    title = models.CharField("title", null=True)
```

Comment:
Look, in the `User` model you have `Birthday` (in description). You should follow the same style and fix the description here:
```suggestion
    title = models.CharField("Title", null=True)
```

Moreover, the description by default will be the same, so you can just remove it